vite.config.ts

```diff
@@ -28,6 +28,28 @@ export default createAppConfig({
 						if (id.includes('vue-material-design-icon')) {
 							return 'icons'
 						}
+						if (id.toLowerCase().includes('album')) {
+							return 'albums'
+						}
+						if (id.toLowerCase().includes('folder')) {
+							return 'folders'
+						}
+						if (id.toLowerCase().includes('place')) {
+							return 'places'
+						}
+						if (id.toLowerCase().includes('tag')) {
+							return 'tags'
+						}
+						if (id.toLowerCase().includes('face')) {
+							return 'faces'
+						}
+						if (id.toLowerCase().includes('sidebar') || id.toLowerCase().includes('tab') || id.toLowerCase().includes('leaflet')) {
+							return 'photos-sidebar'
+						}
+						if (id.toLowerCase().includes('node_modules')) {
+							return 'vendor'
+						}
```

Comment:
Especially this I would consider not that good as then you do not even need to create any different entry points at all. Because then you again bundle everything in one chunk even if it is never used by the entry point.